cineast-api/build.gradle

```diff
@@ -17,6 +17,21 @@ distributions {
     }
 }
 
+configurations.all {
+    // Check for updates every build
+    resolutionStrategy.cacheChangingModulesFor 0, 'seconds'
+}
+
+repositories {
+    maven {
+        url "https://packagecloud.io/manusant/beerRepo/maven2"
+    }
+    maven {
+        // DBIS Nexus
+        url "https://dbis-nexus.dmi.unibas.ch/repository/maven2/"
```

Comment:
Why is this needed?

Reply:
To have our `Spark-Swagger` fork available

Reply:
As long as the build depends on internal repositories, we cannot publish it. What is part of this fork? Is it really necessary to have a fork, can't the modifications be made on the cineast side?

Reply:
To add on luca's point, i compared the changes from your fork to master and i don't see why we should not try to get a PR merged upstream.